### ISSUE TITLE
Discord rich presence is not working

### ISSUE THREAD

**Issue by shadowpanther** [Yellow-Dog-Man/Resonite-Issues#646]
### Describe the bug?

Resonite fails to initialise Discord connection and Discord does not show rich presence because of this.
Looks like the `discord_game_sdk` library is missing from the normal Windows build.

### To Reproduce

Launch Resonite, look at your status in Discord, it just says you are playing Resonite, without extra details

### Expected behavior

Rich presence working

### Resonite Version Number

2023.11.11.1231

### What Platforms does this occur on?

Windows

### Log Files

[VIXEN - 2023.11.11.1231 - 2023-11-12 03_37_44.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13327014/VIXEN.-.2023.11.11.1231.-.2023-11-12.03_37_44.log)
Relevant part:
```
03:37:53.032 ( -1 FPS)		Discord
03:37:53.056 ( -1 FPS)	Exception initializing connector: FrooxEngine.Interfacing.DiscordConnector
System.DllNotFoundException: discord_game_sdk
  at (wrapper managed-to-native) DiscordGameSDKWrapper.Discord.DiscordCreate(uint,DiscordGameSDKWrapper.Discord/FFICreateParams&,intptr&)
  at DiscordGameSDKWrapper.Discord..ctor (System.Int64 clientId, System.UInt64 flags) [0x00309] in <b7bc4445f67b4b1f98d8bf0545869644>:0 
  at FrooxEngine.Interfacing.DiscordConnector+<Initialize>d__26.MoveNext () [0x00029] in <b0765a94dd294f17801ac033e8a2e328>:0 
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x0003e] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x00028] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x00008] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.Runtime.CompilerServices.TaskAwaiter`1[TResult].GetResult () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at FrooxEngine.PlatformInterface+<Initialize>d__12.MoveNext () [0x000e4] in <b0765a94dd294f17801ac033e8a2e328>:0 
```

### Reporters

Shadow Panther

**Comment by ProbablePrime** [comment]
Hey @Frooxius , It seems the build doesn't have this new DLL in it. I updated build scripts where I could find them but it seems those aren't working out. I'm a little uncertain about your directory setup as you run builds though so please take a look!

**Comment by ProbablePrime** [comment]
Somewhat related to #640 too.

**Comment by ProbablePrime** [comment]
For some added context for froox: discord_game_sdk.dll found in https://github.com/Yellow-Dog-Man/FrooxEngine/blob/master/FrooxEngine/discord_game_sdk.dll should be copied to the unity area, I think it needs to go into "plugins" or whatever.

**Comment by Frooxius** [comment]
This should be fixed in 2023.11.15.1100. Thanks for the report!
